### PR TITLE
Create models/data classes for secure conversation welcome screen

### PR DESCRIPTION
**Pull request by DavDo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1894

MOB 1894

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/RemoteConfiguration.kt [hunk 1]
```diff
@@ -26,14 +27,18 @@ internal data class RemoteConfiguration(
     val alertRemoteConfig: AlertRemoteConfig?,
 
     @SerializedName("endScreenSharing")
-    val endScreenShareRemoteConfig: EndScreenShareRemoteConfig?
+    val endScreenShareRemoteConfig: EndScreenShareRemoteConfig?,
```
Comment (gugalo) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/RemoteConfiguration.kt:30]: FYI: you will probably have merge conflicts. In another PR (should be already merged) moved the `endScreenShareRemoteConfig` under `callVisualizerRemoteConfig`

**Review by gugalo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/RemoteConfiguration.kt (hunk 1 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/RemoteConfiguration.kt:30]: Thank you for informing

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/secureconversations/SecureConversationsWelcomeScreenConfig.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    val disabledSendButtonRemoteConfig: ButtonRemoteConfig?,
+    @SerializedName("loadingSendButton")
+    val loadingSendButtonRemoteConfig: ButtonRemoteConfig?,
+    @SerializedName("activityIndicatorColor")
+    val activityIndicatorColorLayerRemoteConfig: ColorLayerRemoteConfig?,
+@SerializedName("messageWarning")
+    val messageWarningRemoteConfig: TextRemoteConfig?,
+@SerializedName("messageWarningIconColor")
+    val messageWarningIconColorLayerRemoteConfig: ColorLayerRemoteConfig?,
+@SerializedName("filePickerButton")
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/secureconversations/SecureConversationsWelcomeScreenConfig.kt:46]: NIT. Looks like several `@SerializedName...` lines have a different indentation.

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/secureconversations/SecureConversationsWelcomeScreenConfig.kt (hunk 2 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/config/secureconversations/SecureConversationsWelcomeScreenConfig.kt]: fixed 3f5e086aa130f5fd5ee5b3e411acde488caa7c63

**Pull request merged**